index.html

```diff
@@ -340,6 +341,21 @@ <h1>About <br>
             </ul>
           </div>
         </section>
-    </main>
+        </main>
+        <section class="contact-me">
+          <footer>
+            <div class="blue-rounded">
+              <img src="assets/contact-form.svg" alt="Contact form background image">
+              <h2 class="contact-heading">Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+              <form action="https://formspree.io/f/xgedddaq" method="post">
+                <input id="name" type="text" name="name" value="Yeremias"><br>
+                <input id="email" type="email" name="email" value="natajayanj14@gmail.com"><br>
```

Comment:
Please ensure that all input fields have the correct associated placeholder text according to the project requirement.